qiskit_experiments/characterization/t2ramsey_analysis.py

```diff
@@ -122,25 +127,24 @@ def _format_plot(ax, unit, fit_result, conversion_factor):
             figures = None
 
         # Output unit is 'sec', regardless of the unit used in the input
-        analysis_result = AnalysisResult(
-            {
-                "t2ramsey_value": fit_result["popt"][1],
-                "frequency_value": fit_result["popt"][2],
-                "stderr_t2": fit_result["popt_err"][1],
-                "stderr_freq": fit_result["popt_err"][2],
-                "unit": "s",
-                "label": "T2Ramsey",
-                "fit": fit_result,
-                "quality": self._fit_quality(
-                    fit_result["popt"], fit_result["popt_err"], fit_result["reduced_chisq"]
-                ),
-            }
-        )
-
-        analysis_result["fit"]["circuit_unit"] = unit
+        result_data = {
+            "t2ramsey_value": fit_result["popt"][1],
```

Comment:
For a follow up PR. But I think this needs to be changed into two analysis results if you want to show both T1 value and frequency value in DB.